src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -178,7 +178,7 @@ const badge = {
     indicatorColor: colors.purple,
   },
   new: {
-    background: `linear-gradient(90deg, ${colors.green}, ${colors.greenDark})`,
+    background: `linear-gradient(90deg, ${colors.green}, ${colors.green500})`,
```

Comment:
where does `green` fit on the scale?

Reply:
`green` -> `green400`

Reply:
Good catch though. I’ll go through the colors one more time in cased I goofed up any others.